backend/app/user/service.py

```
@@ -18,14 +18,21 @@
             user_id = str(user["_id"])
         except Exception:
             return None  # Handle invalid ObjectId gracefully
+        # Ensure ISO 8601 string for dates
+        def iso(dt):
```

Flake8 (7.2.0)

[error] expected 1 blank line before a nested definition, found 0

(E306)